oozie-to-airflow/converter/oozie_converter.py

```diff
@@ -1,4 +1,5 @@
-# Copyright 2018 Google LLC
+# -*- coding: utf-8 -*-
```

Comment:
Why are we changing the name back to `oozie_converter.py`? This way we have 2 files in the project with the same name.
I believe `converter.py` is ok.